src/api/app/views/webui2/webui/kiwi/images/edit.html.haml

```diff
@@ -0,0 +1,38 @@
+- @layouttype = 'custom'
+- @package = @image.package
+= content_for :ready_function do
```

Comment:
Can you please put the javascript after all the Ruby code? (so at the end of the file)